dev-cpp/opentelemetry-cpp/opentelemetry-cpp-1.6.0-r3.ebuild

```diff
@@ -51,6 +51,15 @@ src_configure() {
 	cmake_src_configure
 }
 
+src_install() {
```

Comment:
We should define phases in execution order, so `src_install` should be after `src_test`. I'll fix it on merging.